tools/build-changelog.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/bash
-cat CHANGELOG.md \
-  | sed -E 's|([^/ ]+/[^#]+)#([0-9]+)|[\1#\2](https://github.com/\1/issues/\2)|g' \
-  | sed -E 's| #([0-9]+)| [#\1](https://github.com/joyent/node/issues/\1)|g' \
-  | sed -E 's|([0-9]+\.[0-9]+\.[0-9]+),? Version ([0-9]+\.[0-9]+\.[0-9]+)|<a id="v\2"></a>\
-# \1 Version \2|g' \
-  | sed -E 's|(,? ?)([0-9a-g]{6})[0-9a-g]{34}|\1[\2](https://github.com/joyent/node/commit/\2)|g' \
-  | ./node tools/doc/node_modules/.bin/marked > out/doc/changelog-body.html
-
-cat doc/changelog-head.html \
-    out/doc/changelog-body.html \
-    doc/changelog-foot.html \
-  | sed -E 's|__VERSION__|v'$(python tools/getnodeversion.py)'|g' \
-  > out/doc/changelog.html
-
-rm out/doc/changelog-body.html
```

Comment:
nada, that stuff was for changelog.html, which we don't publish (yet); we just point people to CHANGELOG.md currently.

When we come up with a better strategy we can fix this, however I think this might be something that the @iojs/website team takes responsibility for building something for coming up wtih something much prettier for.

For details on how to build the CHANGELOG.md entry itself, see doc/releases.md above.